### ISSUE TITLE
Strange Behaviour With New Config Options in Version 1.1.0

### ISSUE THREAD

**Issue by mamatt** [mamatt/files_readmemd#28]
In GitLab by @gohrner on Jul 4, 2019, 21:07

I just upgraded from 1.0.4.something to the official 1.1.0 from the app store.

Afterwards, the README.md-Content did not have the "README.md" header any more, and the background was white instead of yellow.

It was ok not to have the headline, but I preferred the yellow background, so I enabled it in the settings - and the app stopped working and did neither show HEADER nor README any more. (As before, I tested it with another user different from the NX admin.)

I disabled the yellow background again and the texts were rendered again.

To debug this, I played a bit with the config settings, but unfortunately forgot / thought too late about opening the javascript console in my browser.

Long story short, this misbehaviour was reproducable for a while, but since I played around "sufficiently" with the config settings, the app suddenly worked as expected.

Could it be that all / most of the config settings currently need to be toggled manually before the app works properly, and bails out after a new installation if the "yellow background" check box is activated?
